mobile/lib/features/trade/trade_dialog.dart

```diff
@@ -125,3 +164,47 @@ Future<void> shareTweet(PositionAction action) async {
     await Share.share(shareText);
   }
 }
+
+class ClickableHelpText extends StatelessWidget {
+  const ClickableHelpText({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return RichText(
+      text: TextSpan(
+        text: 'Please help us fix this issue and join our telegram group: ',
```

Comment:
```suggestion
        text: 'If the issue persists, please report it in our Telegram group, on GitHub or send us a Twitter DM.',
```

Maybe we could use hyperlinks too.

Reply:
I rather pipe them all into one channel.